### DISCUSSION TITLE
(android) building v from source will result an error

### DISCUSSION THREAD

**Discussion by spiderites** [vlang/v#15874] [Installation Issues]
running make will result 
```
 ./vc/v.c:21092:11: error: incompatible integer to pointer conversion initializing 'char **' with an expression of type 'int' [-Wint-conversion]
                        char** csymbols = backtrace_symbols(((voidptr)(&buffer[v_fixed_index(skipframes, 100)])), nr_actual_frames);
                               ^          ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.
make: *** [GNUmakefile:108: all] Error 1 
```

**Comment by hungrybluedev** [comment]
Can we have more details about where you're running the code? Perhaps paste the output of `v doctor`.

**Reply by spiderites** [reply]
As I could not install v,  I cannot do that.

**Comment by spytheman** [comment]
What is your platform? It does work on Termux.

**Reply by spiderites** [reply]
yes, I'm using termux.
I tried building it on my computer it worked fine. I also tried doing it on my phone it worked. However I deleted a lot of stuffs on my phone and that included termux. So i downloaded termux (from f droid) again and installed v a few days later, this error show up. 
Im on android 8 (no root)

**Reply by spiderites** [reply]
I also have noticed that termux updated from clang 14 > clang 15

**Comment by islonely** [comment]
I'm getting the same issue among other errors. Edit: installing TCC and changing `cc` flag in `GNUmakefile` to point to TCC produces this error when trying to compile:
```
CANNOT LINK EXECUTABLE "./v1.exe": cannot locate symbol "tcc_backtrace" referenced by "/data/data/com.termux/files/home/v/v1.exe"...
make: *** [GNUmakefile:114: all] Error 1
```

![Screenshot_2023-09-10-07-30-04-66_fc704e6b13c4fb26bf5e411f75da84f2](https://github.com/vlang/v/assets/31167933/cedad079-513b-4181-b529-1989fae01cdc)

![Screenshot_2023-09-10-07-32-55-30_84d3000e3f4017145260f7618db1d683](https://github.com/vlang/v/assets/31167933/10360a5d-c662-4d3e-aeb5-1e83c287b8a7)
![Screenshot_2023-09-10-07-30-13-25_fc704e6b13c4fb26bf5e411f75da84f2](https://github.com/vlang/v/assets/31167933/4ce7d311-a20a-45b8-a7db-fa06d265476b)

**Reply by 7underlines** [reply]
Execute this and try again.
```bash
pkg install clang libexecinfo libgc libgc-static make git
```

**Comment by 7underlines** [comment]
Execute this and try again.
```bash
pkg install clang libexecinfo libgc libgc-static make git
```
